test/fixtures/lightscript/if-statement/oneline-return-colons/actual.js

```diff
@@ -0,0 +1,2 @@
+f = ->
+  if true: return 1 else: return 2
```

Comment:
I don't think we need a test for this... was a silly mistake on my part to suggest. 

```
if true: 1 else: 2
```
and/or
```
if true: return 1
else: return 2
```
might make more sense instead. But not sure either of those are necessary